lib/processor.js

```diff
@@ -32,6 +32,12 @@
 			var pxre = /([\d\.]+)\D*/;
 			var width = svgelem.getAttribute( "width" );
 			var height = svgelem.getAttribute( "height" );
+			var viewbox = svgelem.getAttribute( 'viewBox' );
+			if (viewbox && width === '100%' && height === '100%') {
```

Comment:
I agree, I think we should always default to `viewBox` (unless `defaultWidth/defaultHeight` values are specified) but that would be a breaking change, indeed.

For now, in case of a `viewBox` and no width/height specified the script defaults to 400x300 (or whatever the user specified) so exactly as before.